modules/experimental/pages/destiny-not-fate.adoc

```diff
@@ -0,0 +1,16 @@
+= Destiny Not Fate
```

Comment:
```suggestion
= Destiny, Not Fate
```

Reply:
Hmm. Grammatically, it should have the comma (and seems a touch odd without it). What's your beef with it?

It's not a hill I'll choose to die on — but there are other alternatives we might consider, which vary depending on what your core objection is.

Reply:
From a pragmatic point of view, it means redoing the entire way that we display unit notes.

I remember you saying that we should be doing that anyway, in case of other potential ability names containing commas, and to do thing 'properly' I agree. But so far it's one ability name out of about 60.

We _should_ change the display format instead, but that that involves changing every page on the Remastered site that contains a data sheet, updating hundreds of data sheets in the builder's master unit file*, all the detachment files in the builder, and a substantial amount of code. :-(

*And not in a way where I would feel safe using find-and-replace.

Reply:
Okay. If it's too much trouble to fix it the 'right way' where the builder is concerned, then there's a simple compromise we can make:

**We include the comma in the main project, and leave it absent in the builder. Job done.**

Realistically, it won't cause any actual problems and only pedants like us will care about this tiny inconsistency. It's imperfect but pragmatic solution: Perfect is the enemy of good, and all that.